### ISSUE TITLE
Gitstatus fails

### ISSUE THREAD

**Issue by NeonSupernova** [romkatv/gitstatus#284]
Using iSH (alpine linux), I get this error:

[ERROR]: gitstatus failed to initialize.

  Zsh log (/tmp/gitstatus.POWERLEVEL9K.0.5.1639333235.1.xtrace.log):

    +(anon):7> setopt monitor
    +(anon):9> ((  ! _GITSTATUS_STATE_POWERLEVEL9K  ))
    +(anon):10> [[ -r /proc/version && 'Linux version 4.20.69-ish iSH 1.2.3 (295) Dec 11 2021 06:09:11' == *Microsoft* ]]
    +(anon):13> print -rn
    +(anon):14> zsystem flock -f lock_fd /tmp/gitstatus.POWERLEVEL9K.0.5.1639333235.1.lock
    +(anon):15> [[ 15 == <1-> ]]
    +(anon):18> typeset -gi '_GITSTATUS_LOCK_FD_POWERLEVEL9K=lock_fd'
    +(anon):20> [[ linux-musl == cygwin* ]]
    +(anon):41> sysopen -r -o cloexec -u resp_fd /proc/self/fd/14
    (anon):sysopen:41: can't open file /proc/self/fd/14: no such file or directory
    +(anon):41> return
    +(anon):41> _gitstatus_daemon_p9k_
    +_gitstatus_daemon_p9k_:1> local -i pipe_fd

Daemon log (/tmp/gitstatus.POWERLEVEL9K.0.5.1639333235.1.daemon.log):

    +_gitstatus_daemon_p9k_:3> local pgid=42
    +_gitstatus_daemon_p9k_:4> [[ 42 == <1-> ]]
    +_gitstatus_daemon_p9k_:5> cd -q /
    +_gitstatus_daemon_p9k_:90> ((  lock_fd == -1  ))
    +_gitstatus_daemon_p9k_:9> trap '' PIPE
    +_gitstatus_daemon_p9k_:11> local uname_sm
    +_gitstatus_daemon_p9k_:93> zsystem flock -- /tmp/gitstatus.POWERLEVEL9K.0.5.1639333235.1.lock
    _gitstatus_daemon_p9k_:zsystem:93: failed to open /tmp/gitstatus.POWERLEVEL9K.0.5.1639333235.1.lock for writing: no such file or directory
    +_gitstatus_daemon_p9k_:12> uname_sm=+_gitstatus_daemon_p9k_:12> uname -sm

  System information:

    zsh:      5.8
  If you need help, open an issue and attach this whole error message to it:

    https://github.com/romkatv/gitstatus/issues/new
_p9k_worker_start:sysopen:6: can't open file /proc/self/fd/11: no such file or directory
prompt_vcs:17: vcs_info: function definition file not found

**Comment by romkatv** [comment]
https://github.com/romkatv/gitstatus/issues?q=ish

tl;dr: bug in iSH: ish-app/ish#783
